### ISSUE TITLE
Library inconsistency with Gradle

### ISSUE THREAD

**Issue by hoseinit** [sitewhere/sitewhere-android#4]
Hi,
I am using `siteWhereFrameworkLibrary` and `siteWhereMQTTLibrary` as external libraries in my Gradle-based Android Project.
As soon as I do the importing the project compiles successfully but encounters error on Runtime:
```/AndroidRuntime: FATAL EXCEPTION: main
                                                                       Process: com.tavanagroup.traffic, PID: 7515
                                                                       java.lang.NoClassDefFoundError: com.sitewhere.android.mqtt.preferences.MqttServicePreferences$1
                                                                           at com.sitewhere.android.mqtt.preferences.MqttServicePreferences.<clinit>```
and other similar errors

**Comment by derekadams** [comment]
We are in the process of revamping the SiteWhere Android SDK so that it is easier to use and works consistently in Android Studio and other development environments. The Android libraries were initially developed in ADT. The new version of the SDK will be available along with the 1.4.0 release later this month. The example projects will be ported over as well. A new repository has been created for the SDK and can be found [here](https://github.com/sitewhere/sitewhere-android-sdk).

**Comment by hoseinit** [comment]
Thank you so much for the fast answer. So we will be waiting for the SDK to be released and be used easily in Gradle.
